spring-integration-mock/src/main/java/org/springframework/integration/mock/MockIntegration.java

```diff
@@ -19,48 +19,41 @@
 import static org.mockito.BDDMockito.given;
 
 import java.util.ArrayList;
-import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
 
 import org.mockito.Mockito;
 
-import org.springframework.beans.BeansException;
-import org.springframework.beans.DirectFieldAccessor;
-import org.springframework.beans.factory.BeanFactory;
-import org.springframework.beans.factory.BeanFactoryAware;
-import org.springframework.beans.factory.config.ConfigurableListableBeanFactory;
-import org.springframework.context.Lifecycle;
 import org.springframework.integration.core.MessageSource;
-import org.springframework.integration.endpoint.SourcePollingChannelAdapter;
 import org.springframework.messaging.Message;
 import org.springframework.messaging.support.GenericMessage;
-import org.springframework.util.Assert;
 
 /**
+ * The factory for integration specific mock components.
+ *
  * @author Artem Bilan
  *
  * @since 5.0
  */
 public final class MockIntegration {
```

Comment:
I just wondered if this might be SRP violation. But as judged by `MockMvc ` it's OK.